core/src/main/java/dk/alexandra/fresco/lib/lp/LPSolver.java

```diff
@@ -90,12 +91,12 @@ private boolean checkDimensions(LPTableau tableau, Matrix<DRes<SInt>> updateMatr
         enumeratedVariables.add(BigInteger.valueOf(i));
       }
 
-      LPState initialState = new LPState(BigInteger.ZERO, tableau, updateMatrix, null, pivot,
+      LpState initialState = new LpState(BigInteger.ZERO, tableau, updateMatrix, null, pivot,
           enumeratedVariables, initialBasis, pivot);
       return () -> initialState;
     }).whileLoop(state -> !state.terminated(), (seq, state) -> {
       iterations++;
-      if (debugLog) {
+      if (getDebuggable()) {
```

Comment:
In general I see the term "isDebug()" rather than this - could we rename it?

Reply:
Done